leetcode/medium/498._Diagonal_Traverse.py

```diff
@@ -0,0 +1,67 @@
+# Link to problem: https://leetcode.com/problems/diagonal-traverse/description/
+# Loom explaining solution you wrote (5m): TODO
+# Solution:
+# Description
+# Key ideas
+# Reason through the boundary conditions by drawing a bigger square with arrows crossing through them
+# There’s nothing difficult but one needs to be extra careful
+# Difficulty: Medium
+# Code
+
+class Solution:
+    def findDiagonalOrder(self, mat: List[List[int]]) -> List[int]:
+        def next_up_cord(cord):
```

Comment:
Please add typing to all these functions. Both for the arguments and the return values.

Reply:
done